### PR TITLE
creates jenkins template

### PR DESCRIPTION
**Pull request by otaviojava**
https://github.com/platformsh/template-jenkins/pull/1

### CODE REVIEW

File: templates/jenkins/files/.platform.app.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+hooks:
+    build: |
+      export MAVEN_OPTS='-Xmx64m' 
+      wget -q http://mirrors.jenkins.io/war-stable/latest/jenkins.war
+
+# The configuration of app when it is exposed to the web.
+# The memory is just to a standard plan.
+web:
+    commands:
+        start: java -jar -Xmx512m jenkins.war --httpPort=$PORT
```
Comment (Crell) [inline templates/jenkins/files/.platform.app.yaml]: Not something to resolve here, but we really need to figure out how to make the memory size dynamic based on the container size.  This is a ticking time bomb at present.

File: templates/jenkins/files/.platform/local/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/
```
Comment (Crell) [inline templates/jenkins/files/.platform/local/.gitignore]: Local folder should be omitted. :-(

File: templates/jenkins/files/.platform.app.yaml (hunk 1 above)
Comment (otaviojava) [reply templates/jenkins/files/.platform.app.yaml]: Yes, we had this discussion yesterday. We're working on it.

File: templates/jenkins/files/.platform/local/.gitignore (hunk 2 above)
Comment (otaviojava) [reply templates/jenkins/files/.platform/local/.gitignore]: Thank you

**Pull request merged**